cmd/juju/controller/destroy.go

```diff
@@ -119,6 +121,16 @@ WARNING! This command will destroy the %q controller.
 This includes all machines, applications, data and other resources.
 `[1:]
 
+var destroySysMsgWithDetails = `
+WARNING! This command will destroy the %q controller.
+This includes all data and other resources:
```

Comment:
The `all data and other resources` for the most part is included below, so I would combine this line with the line starting `WARNING!` above.
Perhaps something like:
`WARNING! This command with destroy the %q controller and all it's resources`

Reply:
thx, fixed